tappsbt/mock.go

```diff
@@ -747,5 +807,18 @@ func (to *TestVOutput) ToVOutput(t testing.TB) *VOutput {
 		)
 	}
 
+	if len(to.AltLeaves) > 0 {
+		v.AltLeaves = make(
+			[]AltLeafAsset, 0, len(to.AltLeaves),
+		)
+
+		for idx := range to.AltLeaves {
+			v.AltLeaves = append(
```

Comment:
nit: could use index based assignment here and above, then the `make` call also gets short enough to fit a single line.

Reply:
I think the `make` line break was needed before adding the `AltLeafAsset` type alias.